### DISCUSSION TITLE
认证前的问题出错！

### DISCUSSION THREAD

**Discussion by xiangmingya** [natfrp/wiki#263] [使用问题] [answered]
![image](https://user-images.githubusercontent.com/70627786/212580947-33ef6ebd-bb2f-4d87-888d-7b809c293b43.png)
![image](https://user-images.githubusercontent.com/70627786/212581035-e7182fe1-d232-47c4-a39d-040e49845c68.png)
这个问题怎么解决呢？有没有人遇到过？

**Comment by fakeboboliu** [comment, marked as answer]
请问你使用的浏览器是？

**Reply by xiangmingya** [reply]
是一款小众浏览器，星愿！我换一个试试！谢谢

**Reply by xiangmingya** [reply]
解决谢谢
